package.json

```diff
@@ -6,6 +6,7 @@
   "engines": {
     "node": ">=12.0.0 <17.0.0"
   },
+  "main": "main.js",
```

Comment:
A lot of projects use `index.js` as their main script, I wonder if we should follow the same convention?